mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -51,6 +52,25 @@ public ChangeManagerImpl(Vertx vertx, String tenantId) { //NOSONAR
     this.tenantId = TenantTool.calculateTenantId(tenantId);
   }
 
+
+  @Override
+  public void deleteChangeManagerJobExecutions(DeleteJobExecutionsDto entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(c -> {
+      try {
+        jobExecutionService.deleteJobExecutionsByIds(entity.getIds(), tenantId)
+          .map(optionalJobExecution -> !optionalJobExecution.booleanValue()?
+            new BadRequestException(format("JobExecution with id '%s' was not found", entity.getIds())): optionalJobExecution)
+          .map(DeleteChangeManagerJobExecutionsResponse::respond200WithApplicationJson)
+          .map(Response.class::cast)
+          .otherwise(ExceptionHelper::mapExceptionToResponse)
+          .onComplete(asyncResultHandler);
+      } catch (Exception e) {
+        LOGGER.error(getMessage("Failed to get JobExecution by id", e, entity.getIds().get(0)));
```

Comment:
Can rewrite to more simple:
LOGGER.error("Failed to delete Job Executions by ids: {}",  entity.getIds(), e);

Reply:
@SerhiiNosko - PR updated as per comments, Thanks 🙂